src/main/java/com/example/java/test/junior/developer/UserController.java

```diff
@@ -10,20 +9,23 @@
 @RestController
 @RequestMapping("/api")
 public class UserController {
+
+    //відповідає за обробку даних, які вносить юзер
+
     ConcurrentHashMap<Long, User> users = new ConcurrentHashMap<>();
 
     @GetMapping("/{id}")
     public User getUser(@PathVariable Long id) {
         return users.get(id);
     }
 
-    @GetMapping("/")
+    @GetMapping("/users")
     public List<User> getAllUsers() {
         return new ArrayList<User>(users.values());
     }
 
-    @PostMapping("/")
-    public User getUsers(@RequestBody User user) {
+    @PostMapping("/new_user")
+    public User createUser(@RequestBody User user) {
```

Comment:
Don't use entity as a response structure, this is not a good practice.  Create response DTO instead